src/lib/eliom_form_sigs.shared.mli

```diff
@@ -0,0 +1,1034 @@
+type input_type =
+  [ `Url
+  | `Tel
+  | `Text
+  | `Time
+  | `Search
+  | `Password
+  | `Checkbox
+  | `Range
+  | `Radio
+  | `Submit
+  | `Reset
+  | `Number
+  | `Hidden
+  | `Month
+  | `Week
+  | `File
+  | `Email
+  | `Image
+  | `Datetime_local
+  | `Datetime
+  | `Date
+  | `Color
+  | `Button
+  ]
```

Comment:
Good catch ;), I just de-inlined it without noticing. It is used (`Eliom_content_sigs`).